src/preferences/dialog/dlgprefreplaygain.cpp

```diff
@@ -3,24 +3,37 @@
 #include "controlobject.h"
 #include "util/math.h"
 
-#define kConfigKey "[ReplayGain]"
-
-static const int kReplayGainReferenceLUFS = -18;
-
-
-DlgPrefReplayGain::DlgPrefReplayGain(QWidget * parent, UserSettingsPointer  _config)
+namespace {
+const char* kConfigKey = "[ReplayGain]";
```

Comment:
Could you move these to a common file and reference them in teh analyzers and here? Too many Analyze vs Analyse differences make it likely to mess up :).